src/Store/SessionStore.php

```diff
@@ -10,10 +10,15 @@
 class SessionStore implements StoreInterface
 {
     /**
-     * Session base name, if not configured.
+     * Default session base name.
      */
     const BASE_NAME = 'auth0_';
```

Comment:
would this get concatenated anything later? If not, any reason why you chose to add a `_` at the end? Asking because I don't see the construct method receiving a "session last name" parameter

Reply:
That's what it was before this change so I'm just keeping the original value.

Reply:
Here's how it's used:

https://github.com/auth0/auth0-PHP/blob/master/src/Store/SessionStore.php#L120

The original base name creates a double underscore, which is unfortunate, but changing that would be BC